runner-manager/cf-driver/prepare.sh

```diff
@@ -51,11 +51,8 @@ create_temporary_manifest() {
     # Align additional environment variables with YAML at end of source manifest
     local padding="      "
 
-    # Add any CI_SERVICE_x variables populated by start_service()
-    for v in "${!CI_SERVICE_@}"; do
-        echo "${padding}${v}: \"${!v}\"" >>"$TMPMANIFEST"
-    done
-    for v in "${!CI_SERVICE_ID_@}"; do
+    # Add any WSR_SERVICE_x variables populated by start_services()
```

Comment:
[suggestion (nb)] match the comment to what the code is doing

```suggestion
    # Add any WSR_* variables populated by start_services()
```